clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -606,31 +612,30 @@ public void handle(OffsetFetchResponse response, RequestFuture<Map<TopicPartitio
                 TopicPartition tp = entry.getKey();
                 OffsetFetchResponse.PartitionData data = entry.getValue();
                 if (data.hasError()) {
-                    log.debug("Error fetching offset for topic-partition {}: {}", tp, Errors.forCode(data.errorCode)
-                            .exception()
-                            .getMessage());
-                    if (data.errorCode == Errors.GROUP_LOAD_IN_PROGRESS.code()) {
+                    Errors error = Errors.forCode(data.errorCode);
+                    log.debug("Group {} failed to fetch offset for topic-partition {}: {}", groupId, tp, error.message());
```

Comment:
I think we tend to just say `partition` even though we mean `topic partition`. Is there a reason to say topic-partition here?